src/components/lastRestaurants/lastRestaurants.css

```diff
@@ -0,0 +1,61 @@
+.last-restaurants {
+  padding-top: 20px;
+  width: 100%;
+}
+.last-restaurants__title-name {
+  color: #3f3e3e;
+  padding: 20px 0 20px 86px;
```

Comment:
I think this spacing is not cool, we need to define a limiter in the main container of the page so that all content has an equally lateral spacing


like in prod:

<img width="1432" alt="image" src="https://user-images.githubusercontent.com/38358239/169146163-7a7167d2-3cfb-4d4f-95e0-7f0cee69db51.png">

Reply:
really! I created a global class to set the spacing